workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_home_resources.scss

```diff
@@ -0,0 +1,101 @@
+.home-resources {
+	display: grid;
+	gap: 48px;
+	grid-template-columns: repeat(auto-fit, 184px);
+	grid-template-rows: 150px;
+	max-width: 948px;
+	padding: 0 1.3125rem;
+
+	@media (max-width: 1200px) {
+		gap: 24px;
+		grid-template-columns: repeat(2, 184px);
+	}
+
+	@media (max-width: 576px) {
+		grid-template-columns: repeat(1, 184px);
+	}
+
+	.resource {
+		background-color: #fff;
+		border: 1px solid rgba(9, 16, 24, 0.12);
+		border-radius: 0.25rem;
+		box-shadow: 0 6px 15px -6px rgb(9 16 29 / 6%);
+		min-width: 184px;
+		padding: 1.8125rem 1rem;
+		text-decoration: none;
+
+		&:not(.card-disabled):hover {
+			border-bottom: 3px solid $primary;
+			box-shadow: 0 6px 11px 0 rgb(9 16 29 / 9%);
+			text-decoration: none;
+			transform: translateY(-4px);
+		}
+
+		.circles {
+			background-clip: content-box;
+			border-radius: 50%;
+			height: 48px;
+			padding: 1px;
+			width: 48px;
+
+			svg {
+				height: 20px;
+				margin: 0.5rem 0.55rem;
+				width: 20px;
+			}
+		}
+
+		.title {
+			color: $color-neutral-10;
+			letter-spacing: unset;
+			padding-top: 1rem;
+			text-transform: none;
+		}
+	}
+
+	.resource-container {
+		&:nth-of-type(1) {
```

Comment:
Let's remove this nth-of-type CSS - it makes it confusing. We could just include the circle / icon for each element as a complete svg icon. I saw the issue you were seeing when adding these into sprites but you can just remove the mask to fix that: 

```
<svg display=none height=0 version=1.1 width=0 xmlns=http://www.w3.org/2000/svg
	xmlns:xlink=http://www.w3.org/1999/xlink><symbol id="home" viewBox="0 0 48 48" fill="none">
<circle cx="24" cy="24" r="24" fill="#D1ECFA"/>
<circle cx="24" cy="24" r="20" stroke="white"/>
<path d="M22.3315 30.1063V25.9396H25.6648V30.1063C25.6648 30.5646 26.0398 30.9396 26.4981 30.9396H28.9981C29.4565 30.9396 29.8315 30.5646 29.8315 30.1063V24.2729H31.2481C31.6315 24.2729 31.8148 23.7979 31.5231 23.5479L24.5565 17.2729C24.2398 16.9896 23.7565 16.9896 23.4398 17.2729L16.4731 23.5479C16.1898 23.7979 16.3648 24.2729 16.7481 24.2729H18.1648V30.1063C18.1648 30.5646 18.5398 30.9396 18.9981 30.9396H21.4981C21.9565 30.9396 22.3315 30.5646 22.3315 30.1063Z" fill="#1AA0E8"/>
  </symbol></svg>

<svg class="icon">
  <use xlink:href="#home"></use>
</svg>
```

Reply:
I removed the css and added the svgs to the sprite.  It does work when removing the mask tag in the SVG.  Thanks.